### PR TITLE
Update the editors.

### PR DESCRIPTION
**Pull request by jyasskin**
Replace @igrigorik with @clelland, who's more focused on this spec.

@juliatuttle Do you still want to help edit this spec?

@dcreager FYI and feel free to object.

***
<a href="https://pr-preview.s3.amazonaws.com/jyasskin/network-error-logging/pull/116.html" title="Last updated on Sep 3, 2019, 8:32 PM UTC (04e18fe)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/network-error-logging/116/63492fa...jyasskin:04e18fe.html" title="Last updated on Sep 3, 2019, 8:32 PM UTC (04e18fe)">Diff</a>

### CODE REVIEW

**Comment by igrigorik** [comment]
Same as https://github.com/w3c/reporting/pull/175, let's move past editors into former editors block?

**Comment by jyasskin** [comment]
Done.

File: index.html [hunk 1]
```diff
@@ -19,11 +19,10 @@
...
-      name: "Ilya Grigorik",
-      url: "https://www.igvita.com/",
-      mailto: "igrigorik@gmail.com",
+      name: "Ian Clelland",
+      mailto: "iclelland@google.com",
       company: "Google",
-      w3cid: "56102"
+      w3cid: "76841"
     }, {
       name: "Alois Reitbauer",
```
Comment (igrigorik) [inline index.html]: Alois until Oct 2014

File: index.html [hunk 2]
```diff
@@ -34,6 +33,14 @@
       mailto: "juliatuttle@chromium.org",
```
Comment (igrigorik) [inline index.html]: Julia until September 2017

**Comment by dcreager** [comment]
No objections from me!  :+1: pending Ilya's suggested edits.

File: index.html (hunk 2 above)
Comment (jyasskin) [reply index.html]: Done.

File: index.html (hunk 1 above)
Comment (jyasskin) [reply index.html]: Done.

**Pull request merged**